syntheseus/tests/search/algorithms/test_pdvn.py

```diff
@@ -0,0 +1,428 @@
+from __future__ import annotations
+
+import math
+from collections import Counter
+
+import pytest
+
+from syntheseus.search.algorithms.pdvn import PDVN_MCTS, pdvn_extract_training_data
+from syntheseus.search.chem import BackwardReaction, Molecule
+from syntheseus.search.node_evaluation.common import ConstantNodeEvaluator
+from syntheseus.search.reaction_models.toy import ListOfReactionsModel
+from syntheseus.tests.search.algorithms.test_base import BaseAlgorithmTest
+from syntheseus.tests.search.algorithms.test_best_first import (
+    DictMolCost,
+    DictRxnCost,
+    rxn_cost_fn,  # noqa: F401
+)
+from syntheseus.tests.search.conftest import RetrosynthesisTask
+
+
+@pytest.fixture
+def policy(
+    rxn_cost_fn,  # noqa: F811  # doesn't understand pytest's use of rxn_cost_fn as fixture
+) -> DictRxnCost:
+    return DictRxnCost(
+        rxn_smiles_to_cost={rxn: math.exp(-c) for rxn, c in rxn_cost_fn.rxn_to_cost.items()},
+        default=0.4,
+    )
```

Comment:
Huh, weird - is it `flake8` that is complaining about `rxn_cost_fn` being seemingly unused?

Reply:
Yes, that's my understanding. Syntactically it is unused, it just ends up being used by `pytest` but `flake8` doesn't "know" about this.